app/assets/src/components/ui/controls/PlusMinusControl.jsx

```diff
@@ -0,0 +1,35 @@
+import cx from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import PlusControlIcon from "~/components/ui/icons/PlusControlIcon.jsx";
+import MinusControlIcon from "~/components/ui/icons/MinusControlIcon.jsx";
+import cs from "./plus_minus_control.scss";
+
+const PlusMinusControl = ({
+  onClickPlus,
+  onClickMinus,
+  plusDisabled,
+  minusDisabled,
+  className,
+}) => {
+  return (
+    <div className={cx(className, cs.plusMinusContainer)}>
+      <button disabled={plusDisabled} onClick={onClickPlus}>
```

Comment:
also is this styling already changing when 'disabled'?

Reply:
I think the button on its own changes the cursor, but other than that nothing is changed in styling (not too sure what it should look like disabled either)